pkg/types/ovirt/validation/platform.go

```diff
@@ -30,5 +34,9 @@ func ValidatePlatform(p *ovirt.Platform, fldPath *field.Path) field.ErrorList {
 	if p.DefaultMachinePlatform != nil {
 		allErrs = append(allErrs, ValidateMachinePool(p.DefaultMachinePlatform, fldPath.Child("defaultMachinePlatform"))...)
 	}
+	if len(c.ObjectMeta.Name) > maxCharsName {
```

Comment:
Looking back over this again, I am struck that this is not the correct place to check this. The `ValidatePlatform` function is meant to validate the `.platform.ovirt` field of the install config. It is not the ideal place to validate other fields. Might it be better to add a call to a new `ovirtvalidation.ValidateClusterName` function in [this](https://github.com/openshift/installer/blob/master/pkg/types/validation/installconfig.go#L69-L72) area?

Reply:
I do see that the OpenStack validation has set the precedence for doing it the way that you have done it here. I still suggest that we don't copy that pattern here, though.